### ISSUE TITLE
manifest.json is declared twice (one with wrong path) when creating a pwa with Rollup and No Express API server

### ISSUE THREAD

**Issue by creal73** [muenzpraeger/create-lwc-app#626]
### Summary

When generating a new app with **pwa**, **rollup** and **no Express API server** options, `lwc-services.config.js` contains duplicate (and wrong '/client' ) manifest.json resource.

### Steps To Reproduce:

1.  `npx create-lwc-app`
1.  Do you want to use the simple setup? **No**
1. Package name for npm **test**
1. Description
1. Author
1. Select a package manager **npm**
1. Select the type of app you want to create **Progressive Web App (PWA)**
1. Select a bundler **Rollup**
1. Use TypeScript Or JavaScript **JavaScript**
1. Do you want a basic Express API server? **No**

### Expected result

`manifest.json` should be declared once.

### Actual result

```
module.exports = {
    resources: [
        { from: 'src/resources/*', to: 'dist/resources/' },

        { from: 'src/index.html', to: 'dist/' },
        { from: 'src/manifest.json', to: 'dist/' },

        { from: 'src/client/manifest.json', to: 'dist/manifest.json' } <== should not be declared
    ]
};
```

### Additional information

![image](https://user-images.githubusercontent.com/587284/86575047-f410a880-bf76-11ea-888e-e5b97b4de516.png)

![image](https://user-images.githubusercontent.com/587284/86575186-2ae6be80-bf77-11ea-9127-d2369eaf9ba5.png)

**Comment by rochejul** [comment]
+1

**Comment by muenzpraeger** [comment]
Fixed, and published with 2.1.4 - thanks for the finding!
